save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityStatus.kt

```diff
@@ -10,17 +10,17 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 @JsExport
-@Suppress("IDENTIFIER_LENGTH")
+@Suppress("IDENTIFIER_LENGTH", "WRONG_DECLARATIONS_ORDER")
 enum class VulnerabilityStatus(val value: String) {
     /**
-     * Approve status
+     * Automatically approved by batch-upload
      */
-    APPROVED("Approved"),
+    AUTO_APPROVED("Approved"),
```

Comment:
Strange that diktat is not triggering here (APPROVED actually should go first)

Reply:
Diktat triggered here, I added `@Suppress` for this, this order seemed more logical to me. I can bring the old order back if needed.

Reply:
Decided to return old order